package.json

```diff
@@ -25,8 +26,8 @@
   },
   "scripts": {
     "prepare": "husky install",
-    "start": "react-scripts start",
-    "build": "react-scripts build",
+    "start": "react-scripts --max_old_space_size=4096 start",
```

Comment:
Czy to nie jest jakaś forma obejścia tego że mamy memory leak'a?? XDD

Reply:
Szczerze mówiąc nie wiem, na stacku pisali, że tak się to rozwiązuje. Plus zauważ, że to nie koniecznie musi być memory leak - to wywaliło się w testach, czyli aplikacja była świeżo postawiona. Wydaje mi się, że to oznacza tyle, że aplikacja ma za mało pamięci defaulotowo do wstania.